src/components/inbox/Inbox.js

```diff
@@ -94,12 +91,14 @@ class Inbox extends Component {
   };
 
   render() {
-    const { open, inbox, all, close } = this.props;
+    const { open, inbox, all, close, modalVisible } = this.props;
+
     return (
       <div
         className="js-inbox-wrapper"
         onKeyDown={e => this.handleKeyDown(e)}
-        tabIndex={0}
+        onFocus={this.props.onFocus}
```

Comment:
Why not get it from props ? ^^

Reply:
Do you mean `tabIndex`? I just kept the `0` for now, since it's a sensible default and this component is only used in one place. We can add that prop later should we need it

Reply:
Whoopsie, haha, got it